build/charts/antrea/crds/flowexporterdestination.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: flowexporterdestinations.crd.antrea.io
+  labels:
+    app: antrea
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - name: Address:Port
+          type: string
+          description: Address of flow collector.
+          jsonPath: .spec.address
+        - name: Protocol
+          type: string
+          description: Protocol used to send network flows.
+          jsonPath: .spec.protocol
+        - name: Filter
+          type: string
+          description: Filter for network flows, nil means no filter.
+          jsonPath: .spec.filter
+          priority: 10
+        - name: Connected
+          type: boolean
+          description: Status of last connection to flow collector.
+          jsonPath: .status.conditions[?(@.type=="Connected")].status
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              required:
+                - address
+                - protocol
+              properties:
+                address:
+                  type: string
+                  description: >
+                    The flow collector address including port as a string.
+
+                    Example:
+                    - flow-aggregator/flow-aggregator:14739
+                    - 10.244.10.10:4739
+                protocol:
+                  type: object
+                  description: >
+                    The protocol used to send flow details.
+
+                    Exactly one must be defined and non-nil.
+                  x-kubernetes-validations:
+                    - rule: "[has(self.ipfix), has(self.grpc)].filter(x, x).size() == 1"
+                      message: "Exactly one exporter protocol must be specified and non-nil."
```

Comment:
we usually use the built-in `oneOf` here instead of relying on `x-kubernetes-validations`. You can look for examples in other CRDs.

Reply:
Updated!